onnxruntime/core/providers/xnnpack/xnnpack_execution_provider.cc

```diff
@@ -97,8 +99,12 @@ XnnpackExecutionProvider::XnnpackExecutionProvider(const XnnpackExecutionProvide
 
 // implement RegisterAllocator to test/validate sharing the CPU EP's allocator
 void XnnpackExecutionProvider::RegisterAllocator(AllocatorManager& allocator_manager) {
-  OrtDevice cpu_device{OrtDevice::CPU, OrtDevice::MemType::DEFAULT, DEFAULT_CPU_ALLOCATOR_DEVICE_ID};
+  OrtDevice const cpu_device{OrtDevice::CPU, OrtDevice::MemType::DEFAULT, DEFAULT_CPU_ALLOCATOR_DEVICE_ID};
 
```

Comment:
what's the purpose of adding `const` here after the type name?

Reply:
`Clangd` implies it's better to mark it as constant, and I believe it does not affect on where it refenced.

Reply:
Can we please use the 'normal' ordering of 'const OrtDevice'?

Reply:
Sure. Done